### PR TITLE
Link released version of sonar-plugin-api-impl

### PR DESCRIPTION
**Pull request by costin-zaharia-sonarsource**
- the previous one was deleted

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -56,7 +56,7 @@
     <doclint>none</doclint>
     <sonar.analyzer.commons>1.25.0.1003</sonar.analyzer.commons>
     <sonar.version>9.8.0.203</sonar.version>
-    <sonar.api.impl.version>9.6.0.57076</sonar.api.impl.version>
+    <sonar.api.impl.version>9.6.0.58139</sonar.api.impl.version>
```
Comment (pavel-mikula-sonarsource) [inline pom.xml:59]: What makes this version better than the old one? I see 9.6.0.58139 in `sonarsource-public-builds` that is as temporary as the QA AFAICT. QA gets promoted to public builds. Both get deleted after 30 (?) days.

Comment (costin-zaharia-sonarsource) [reply pom.xml:59]: So even the public builds are deleted? Then, I don't know what to use.

Comment (costin-zaharia-sonarsource) [reply pom.xml]: Thanks for spotting this. I got confused and thought that public builds are also fine. 

I've changed the version now to `9.5.0.56709` that is the latest public release.

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**